var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -60,6 +60,7 @@ class Hwloc(AutotoolsPackage):
     depends_on('libpciaccess', when='+pci')
     depends_on('libxml2', when='+libxml2')
     depends_on('pkg-config', type='build')
+    depends_on('libpciaccess', when=(sys.platform != 'darwin'))
```

Comment:
No, it needs to be there... I disabled `libpciaccess` in `hwloc` by making the `pci` variant default to `False`. In hindsight, that's wrong. I will need to correct `hwloc`. This patch here should stay as is.